app/controllers/message_drafts_controller.rb

```diff
@@ -22,6 +22,8 @@ def update
     authorize @message
 
     @message.update_content(message_draft_params)
+
+    flash[:notice] = 'Zmeny boli uložené'
```

Comment:
Toto sa kedy vykona? Nebude to blikat vzdy ked sa uklada obsah?

Reply:
@luciajanikova vies mi ukazat video/screen ze kedy sa to vola a kde sa to zobrazi?

Reply:
https://github.com/user-attachments/assets/de614ad5-3ab1-4e10-87d2-69b26c83e0b9